rpc/client/rpc_test.go

```diff
@@ -441,3 +443,65 @@ func TestTxSearch(t *testing.T) {
 		require.Len(t, result.Txs, 0)
 	}
 }
+
```

Comment:
IMHO a test which tests the error'ing or rather un-expected paths would be great: 
 - calling `SendBatch` without beeing in batch mode
 - calling `CancelBatch` without having anything in the enqueued 

(OK, looks like these are the only cases not covered by the tests 👍)

Reply:
Added some tests now that should cover these paths.